CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+4.70.0
+==========
+### Features
+- Added a .WithAccessTokenSha256ToRefresh() method to AcquireTokenForClientParameterBuilder for Managed Identity resource providers, allowing finer control over token refresh scenarios. [Issue #5111](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/5111)
+
```

Comment:
```suggestion
- Added `TokenCacheNotificationArgs.NoDistributedCacheUseReason` in order to indicate that the configured serialized cache should not be a distributed cache to prevent issues when acquiring tokens. [Issue #5199](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/5199)

```